.env

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
`.env` files should not be pushed to Github, you can create `.env.example` with variables' names and without values